test/records/test_id_formats.py

```diff
@@ -6,16 +6,17 @@
 
 class TestIDFormats(SimpleTestCase):
     def test_valid_formats(self):
+        id_regex = re.compile(IDConverter.regex)
         for label, value in (
             ("longformat", "3717ee38900740728076a61a398fcb84"),
             ("guid", "4d8dae2c-b417-4614-8ed8-924b9b4beeac"),
+            ("guid-ish", "c5f5-0f02-4d03-9b5c-8ec7973794b7"),
```

Comment:
I think these changes should be restored see comment see https://github.com/nationalarchives/ds-search/pull/22#discussion_r1955914503